account_invoice_supplierinfo_update/tests/test_account_invoice_supplierinfo_update.py

```diff
@@ -16,7 +16,14 @@ def setUp(self):
         unit = self.env.ref("uom.product_uom_unit")
         self.currency = self.env.ref("base.GBP")
         self.journal.write({"currency_id": self.currency.id})
-
+        self.account = self.env["account.account"].create(
```

Comment:
Instead of this, do the tests on post-install to be sure there's a chart of accounts installed. In fact, it's not correct to assign a receivable account to the product lines.

Reply:
Thanks for the post-install thing, but you haven't removed yet this account creation + the later assignations. Please do it.